docs/concepts/composition.md

```diff
@@ -161,813 +101,157 @@ spec:
-    Status:                True
-    Type:                  Synced
-  Connection Details:
-    Last Published Time:  2020-05-15T07:08:11Z
-Events:
-  Type    Reason                      Age                    From                                       Message
-  ----    ------                      ----                   ----                                       -------
-  Normal  ConfigureCompositeResource  8m23s                  claim/compositemysqlinstances.example.org  Successfully configured composite resource
-  Normal  BindCompositeResource       8m23s (x7 over 8m23s)  claim/compositemysqlinstances.example.org  Composite resource is not yet ready
-  Normal  BindCompositeResource       4m53s (x4 over 23m)    claim/compositemysqlinstances.example.org  Successfully bound composite resource
-  Normal  PropagateConnectionSecret   4m53s (x4 over 23m)    claim/compositemysqlinstances.example.org  Successfully propagated connection details from composite resource
+    name: my-db-connection-details
 ```
 
-## Current Limitations
-
-At present the below functionality is planned but not yet implemented:
-
-* Compositions are mutable, and updating a composition causes all composite
-  resources that use that composition to be updated accordingly. Revision
-  support is planned per issue [#1481].
-
-Refer to the list of [composition related issues] for an up-to-date list of
-known issues and proposed improvements.
-
-[Current Limitations]: #current-limitations
-[Infrastructure Composition Concepts]: composition-concepts.png
-[structural schemas]: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/#specifying-a-structural-schema
-[Infrastructure Composition Provisioning]: composition-provisioning.png
-[composition related issues]: https://github.com/crossplane/crossplane/labels/composition
-[#1481]: https://github.com/crossplane/crossplane/issues/1481
+> We designed Crossplane to be a tool platform teams can use to offer
+> opinionated platform APIs to the application teams they support. The platform
+> team offers those APIs using claims. It helps to think of the claim as an
+> application team’s interface to a composite resource. You could also think of
+> claims as the public (app team) facing part of the opinionated platform API,
+> while composite resources are the private (platform team) facing part.
+
+There are three key differences between an XR and a claim:
+
+1. Claims are namespaced, while XRs (and Managed Resources) are cluster scoped.
+1. Claims are of a different `kind` than the XR - by convention the XR's `kind`
+   without the proceeding `X`. For example a `PostgreSQLInstance` claims an
+   `XPostgreSQLInstance`.
+1. An active claim contains a reference to its corresponding XR, while an XR
+   contains both a reference to the claim an array of references to the managed
+   resources it composes.
+
+![Diagram showing the relationship between claims and XRs][claims-and-xrs]
```

Comment:
Why are there two XPostgreSQLInstances but only one XNetwork?

Reply:
Oh. It belongs to the scenario described below. I wonder how we could make this more clear. E.g. move the diagram to the bottom?

Reply:
It does illustrate that scenario, but I hope it can be a broader demonstration of claims. I experimented with a few places to position the diagram but feel best about where it is. Generally I like the flow of diagram first, then text that explains the diagram. I've added some more labels to this one that hopefully makes what is going on a little clearer. Let me know whether they help.

Reply:
Regarding the position of the diagram: LGTM
The diagram transports a lot of information now. I think it would be helpful to remove certain aspects. e.g. the responsibilities. And leave that to the explanation in the text.

Reply:
Makes sense - done.